src/pages/Autolending/WhoYoullSupportText.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+	<div>
```

Comment:
See AUTO-56. Theres a lot of logic to display that nicely formatted sentence about who you'll lend to.